src/material/table/_table-flex-styles.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * Flex-based table structure
+ */
+$mat-header-row-height: 56px;
+$mat-row-height: 48px;
+$mat-row-horizontal-padding: 24px;
+
+@mixin _mat-table-flex-styles {
```

Comment:
Can you rename this mixin not to have an underscore in its name? With the Sass module system something prefixed with an underscore is considered private and isn't available in other files. The convention we've used in other places is to prefix it with `mat-private-` instead.